### PR TITLE
ebv schema changes

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[4D-Dataset laden](https://trello.com/c/aABW7Bs0/582-4d-dataset-laden)

File: services/src/datasets/external/netcdfcf/mod.rs [hunk 1]
```diff
@@ -851,28 +860,54 @@ fn parse_time_coverage(
...
+            .context(error::CannotConvertTimeCoverageToInt)?;
+        let end = end
+            .parse::<i32>()
+            .context(error::CannotConvertTimeCoverageToInt)?;
+        (
+            NaiveDate::from_ymd(start, 1, 1),
+            NaiveDate::from_ymd(end, 1, 1),
+        )
+    };
+    // add one year to provide a right side boundary for the close-open interval
```
Comment (michaelmattig) [inline services/src/datasets/external/netcdfcf/mod.rs:883]: does this also work for sub-annual datasets? or do those just not exist?

**Review by michaelmattig — DISMISSED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Comment by michaelmattig** [comment]
bors merge

**Review by michaelmattig — APPROVED** [review]

**Comment by michaelmattig** [comment]
bors merge

**Pull request merged**